src/models.py

```diff
@@ -83,6 +83,11 @@ def get_relevant_baselines(task_name):
             (XGBoostModel, {}),
             (AveragingModel, {}),
         ],
+        "kalman_filter": [
+            (LeastSquaresModel, {}),
+            (NNModel, {"n_neighbors": 3}),
+            (AveragingModel, {}),
+        ]
```

Comment:
I notice there isn't an entry for `"noisy_kalman_filter"`. Is this intentional? If so why?

Reply:
no, I was just being an idiot.